main.py

```diff
@@ -0,0 +1,11 @@
+from Lab9.Figures import Circle, Triangle, Cylinder, Cone, Sphere
+
+sasha = Circle("Sasha", "2D", 3, True)
+print(sasha.print_circle())
+anton = Triangle("Anton", "2D", 1, False)
+print(anton.print_triangle())
+anya = Cylinder("Anya", "3D", 9, 3.12)
+print(anya.print_cylinder())
+illya = Cone("Illya", "3D", 3, True)
+print(illya.print_cone())
+tanya = Sphere("Tanya", "3D", 1, True)
```

Comment:
It's better to create method 'print' in class 'GeometricFigures' and override it in every subclass, instead of print_figure() for each figure.

Reply:
Thanks for the advice.Done:)